src/careamics/models/lvae/layers.py

```diff
@@ -117,7 +117,7 @@ def __init__(
                     bias=conv2d_bias,
                 )
                 modules.append(conv)
-                modules.append(nonlin())
+                modules.append(nonlin)
```

Comment:
if `nonlin=nn.ReLU` as docstring suggests then this should be `module.append(nonlin())`

Reply:
Ok nvm, I see the `get_activation` function returns for example `nn.ReLU()`. In which case the docstring should be updated.